blazor/datagrid/connecting-to-data-source/microsoft-sql-server.md

```diff
@@ -0,0 +1,641 @@
+![image](https://github.com/syncfusion-content/blazor-docs/assets/30143756/f653125d-221f-442a-b3a4-13a52df7cf12)---
+layout: post
+title: Microsoft SQL Data Binding in Blazor DataGrid Component | Syncfusion
+description: Learn about consuming data from SQL Server using Microsoft SQL Client, binding it to Syncfusion Component, and performing CRUD operations
+platform: Blazor
+control: DataGrid
+documentation: ug
+---
+
+# Microsoft SQL Server Data Binding
+
+This section describes how to use [System.Data.SqlClient](https://www.nuget.org/packages/System.Data.SqlClient/4.8.6?_src=template) to retrieve data from a Microsoft SQL server database and bind it to the Blazor DataGrid.
+
```

Comment:
Need to add the content related to why we are using custom adpator and url adaptor in this case